tbx/work/models.py

```diff
@@ -21,16 +20,8 @@
 from wagtail.signals import page_published
 
 
-# Currently hidden. These were used in the past and may be used again in the future
-class WorkPageTagSelect(Orderable):
-    page = ParentalKey("work.WorkPage", related_name="tags")
-    tag = models.ForeignKey(
-        "torchbox.Tag", on_delete=models.CASCADE, related_name="work_page_tag_select"
-    )
-
-
-class WorkPageScreenshot(Orderable):
-    page = ParentalKey("work.WorkPage", related_name="screenshots")
+class HistoricalWorkPageScreenshot(Orderable):
+    page = ParentalKey("work.HistoricalWorkPage", related_name="screenshots")
```

Comment:
is this screenshots something that is actually in use somewhere? I can't see the option to add them when editing the page

Reply:
Yes it is, here's a screenshot:

<details><summary>Screenshot</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/d36b0814-ac84-4bb7-9f38-1a20970d4cbf)

</details>

Reply:
It's not displayed in the template anywhere, and wasn't in the previous site either.

Reply:
Oh ok, maybe it's something that was used a long time ago and was later changed

Reply:
I wouldn't let this block merging - but we might need to check in with Lisa if this should be displayed, and whether it should be on the new work page.